lib/api.py

```diff
@@ -27,7 +28,12 @@ class APIv1(object):
     # E: Instance of 'LookupDict' has no 'created' member (no-member)
     def __init__(self, config):
         self.config = config
-        self.host = config.get('host')
+        try:
+            self.host = config.get('host')
+        except ConfigError as error:
+            msg = "To fix this problem define a host in the configuration."
```

Comment:
even better: 

msg = "Missing key from configuration, {0}. Please check your configuration.".format(error)

it will output:

"Execution Error: Missing key from configuration, No option 'host' in section: 'general'. Please check your configuration"